PluralKit.Bot/Commands/MiscCommands.cs

```diff
@@ -43,10 +43,11 @@ public async Task Stats()
         {
             var messagesReceived = Metrics.Snapshot.GetForContext("Bot").Meters.First(m => m.MultidimensionalName == BotMetrics.MessagesReceived.Name).Value;
             var messagesProxied = Metrics.Snapshot.GetForContext("Bot").Meters.First(m => m.MultidimensionalName == BotMetrics.MessagesProxied.Name).Value;
-            
+
             var commandsRun = Metrics.Snapshot.GetForContext("Bot").Meters.First(m => m.MultidimensionalName == BotMetrics.CommandsRun.Name).Value;
-            
+
             await Context.Channel.SendMessageAsync(embed: new EmbedBuilder()
+                .AddField("Connection latency", $"{Context.Client.Latency}ms")
```

Comment:
`DiscordShardedClient.Latency` returns average latency across all shards, perhaps it'd be more useful to display both that and the current shard's latency? (eg. `Client.GetShardFor(Context.Guild).Latency` - make sure to check if it's actually *in* a guild context)

Reply:
Resolved with 11bb41c

Reply:
Finalized with 8a74938